### DISCUSSION TITLE
Import data between osu!lazer installations

### DISCUSSION THREAD

**Discussion by Sarisan** [ppy/osu#23305] [Ideas]
I know how to import data between osu! stable installations, I just copy Songs, Skins, etc folders and db files. But how can I do it with osu!lazer?

**Comment by monochrome22** [comment]
Copy the "osu" folder from Appdata/Roaming. You can also open it through settings "Open osu! folder"

**Reply by Sarisan** [reply]
> Copy the "osu" folder from Appdata/Roaming. You can also open it through settings "Open osu! folder"

But what if my lazer installation is broken and copying folder will just make no difference? My installation is not actually broken, but I'd like to see ability to import data from other lazer installation. Also in cases when you play on Linux and Windows in different time, would be helpful just merge data of those 2 installations.

**Reply by Theighlin** [reply]
If the installation is somehow broken, in stable you can just copy songs, skins etc, but in lazer it will remain broken, there is no such ability.

Idk about "merging" but you can probably have windows and linux share the same data folder (i reckon others have it working, though it is not officially supported)

**Comment by itm154** [comment]
Copying the "files" folder and client.realm file restored all of my data (maps,skins,...etc [excluding login info IIRC]) when my game was broken.
